install/swiftly-install.sh

```diff
@@ -1,716 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
A bit behind I think, why did we remove all of these scripts + their tests?

Reply:
This bash script was the original `curl | bash` installation system for swiftly in the previous version. We've slowly migrated it to the new `swiftly init` mechanism for a few reasons, such as much better options handling, and a generally safer mechanism.

I'm removing it here because it's vestigial at this point, and confuses new contributors who still see this as the installer. Plus, it would need to have support for these new platforms.

Reply:
But it was also responsible for pulling the latest release binary of swiftly right? So is the implication that going forward we should expect users to build from source? Or should we host release binaries on the repo? Just wondering if the readme/release process needs to be changed here

Reply:
There's a few possibilities:
* Direct download of the binary package from swift.org (tarball and macOS pkg) from the downloads page (see https://github.com/swiftlang/swift-org-website/pull/830 for the current draft)
* Reliable API or download URL where the package can be downloaded from swift.org
* System package managers (apt, yum, brew)

Reply:
This should be done in a separate PR to discuss is rather than bundling it in as part of this PR